ServerCore/Pages/Events/Edit.cshtml

```diff
@@ -125,24 +125,24 @@
                 </div>
             </div>
             <div class="form-group">
-                <label asp-for="Event.LockoutIncorrectGuessLimit" class="control-label"></label>
-                <input asp-for="Event.LockoutIncorrectGuessLimit" class="form-control" />
-                <span asp-validation-for="Event.LockoutIncorrectGuessLimit" class="text-danger"></span>
+                <label asp-for="EditableEvent.LockoutIncorrectGuessLimit" class="control-label"></label>
+                <input asp-for="EditableEvent.LockoutIncorrectGuessLimit" class="form-control" />
+                <span asp-validation-for="EditableEvent.LockoutIncorrectGuessLimit" class="text-danger"></span>
             </div>
             <div class="form-group">
-                <label asp-for="Event.LockoutIncorrectGuessPeriod" class="control-label"></label>
-                <input asp-for="Event.LockoutIncorrectGuessPeriod" class="form-control" />
-                <span asp-validation-for="Event.LockoutIncorrectGuessPeriod" class="text-danger"></span>
+                <label asp-for="EditableEvent.LockoutIncorrectGuessPeriod" class="control-label"></label>
+                <input asp-for="EditableEvent.LockoutIncorrectGuessPeriod" class="form-control" />
+                <span asp-validation-for="EditableEvent.LockoutIncorrectGuessPeriod" class="text-danger"></span>
             </div>
             <div class="form-group">
-                <label asp-for="Event.LockoutDurationMultiplier" class="control-label"></label>
-                <input asp-for="Event.LockoutDurationMultiplier" class="form-control" />
-                <span asp-validation-for="Event.LockoutDurationMultiplier" class="text-danger"></span>
+                <label asp-for="EditableEvent.LockoutDurationMultiplier" class="control-label"></label>
+                <input asp-for="EditableEvent.LockoutDurationMultiplier" class="form-control" />
+                <span asp-validation-for="EditableEvent.LockoutDurationMultiplier" class="text-danger"></span>
             </div>
             <div class="form-group">
-                <label asp-for="Event.MaxSubmissionCount" class="control-label"></label>
-                <input asp-for="Event.MaxSubmissionCount" class="form-control" />
-                <span asp-validation-for="Event.MaxSubmissionCount" class="text-danger"></span>
+                <label asp-for="EditableEvent.MaxSubmissionCount" class="control-label"></label>
```

Comment:
Does this actually change the defaults? The name switched, but I'm not seeing the corresponding place where the value changed

Reply:
No, the defaults were actually reasonable, but the first edit to the event properties would whack them.